.github/renovate.json

```diff
@@ -106,12 +106,45 @@
       "matchManagers": ["npm", "nvm"],
       "additionalBranchPrefix": "fe-"
     },
+    {
+      "description": "Exclude frontend deps until Optimize migrates to react-router v6 and react v18",
+      "matchManagers": ["npm", "nvm"],
+      "matchFileNames": ["optimize/**"],
+      "matchPackageNames": [
+        "@carbon/react",
+        "old-bpmn-js",
+        "react",
+        "react-dom",
+        "react-router",
+        "react-router-dom",
+        "@types/react",
+        "@types/react-dom",
+        "@types/react-router-dom"
+      ],
+      "enabled": false
+    },
+    {
+      "description": "Exclude Optimize backend dependencies",
+      "matchManagers": ["maven"],
+      "matchFileNames": ["optimize/**"],
+      "matchPackageNames": [
+        "io.camunda.optimize:optimize-data-generator",
```

Comment:
actually pretty sure this doesn't need to be here either. But for the sake of speed, happy for you to proceed and we clean this up later. I don't think it hurts in any way having it here